### ISSUE TITLE
AngularJS-Error: Cannot read property 'Search' of undefined

### ISSUE THREAD

**Issue by fanningert** [ziahamza/webui-aria2#437]
Current master version (2018.02.18)

I think there is a problem with the last commits. I also found this old issue with the same error (#251).

```
Failed to instantiate module webui due to:
TypeError: Cannot read property 'Search' of undefined
    at mergeTranslation (http://example.com/js/init.js:22:19)
    at http://example.com/js/init.js:46:30
    at Object.invoke (http://example.com/js/libs/angular.min.js:43:175)
    at d (http://example.com/js/libs/angular.min.js:41:64)
    at http://example.com/js/libs/angular.min.js:41:188
    at q (http://example.com/js/libs/angular.min.js:7:351)
    at g (http://example.com/js/libs/angular.min.js:40:488)
    at eb (http://example.com/js/libs/angular.min.js:45:56)
    at c (http://example.com/js/libs/angular.min.js:21:19)
    at Object.Lc [as bootstrap] (http://example.com/js/libs/angular.min.js:21:332
```

**Comment by whgreate** [comment]
same problem ..

**Comment by rafeyu** [comment]
It'll be fixed with this commit #436. I'm sorry ...
